packages/component/src/SendBox/ConnectivityStatus.js

```diff
@@ -10,18 +10,30 @@ import WarningNotificationIcon from '../Attachment/Assets/WarningNotificationIco
 
 const CONNECTIVITY_STATUS_DEBOUNCE = 400;
 
+function setTimeoutSync(fn, interval) {
+  if (interval > 0) {
+    return setTimeout(fn, interval);
+  }
+
+  fn();
+}
+
 const DebouncedConnectivityStatus = ({ interval, children: propsChildren }) => {
-  const [children, setChildren] = useState(propsChildren);
+  const [children, setChildren] = useState(() => propsChildren);
```

Comment:
Can you explain the difference between `useState(() => propsChildren)` and `useState(propsChildren)`? I don't understand the difference.

Reply:
(I believe) the return prevents it from rendering every time - just when there's a change in value.

Reply:
because I definitely HAVEN'T made this mistake 5 billion times

Reply:
Explained offline. 😉